### DISCUSSION TITLE
Are these warnings any cause for concern?

### DISCUSSION THREAD

**Discussion by djnield** [Taxel/PlexTraktSync#610] [Q&A] [answered]
I'm running it locally on Mac using python for a remote server and notice a warning for TV shows and it seems prolific.

`WARNING: <Episode:319511:Homeland-s01e13>: Skipping guid local://319511 because 
provider local has no external Id`

_Is this something I need to be trying to fix?_

In comparison the movie libraries complete in mere minutes stating `[150 it/s]` whereas TV goes for on for half a day `[1 it/s]` or `[0 it/s]`

**Comment by glensc** [comment, marked as answer]
One problem at a time, so I'm going to answer only the first one as it matches the discussion title.

The "Skipping guid local://319511 because provider local has no external Id" means that media item 319511 does not have match from plex agent. you should open the item in plex web and use "Fix Match" to try to resolve the problem.

You can use `plextraktsync inspect 319511` to get a clickable link, or just modify URL manually if you open any item in plex web.

Skipped items can't be synced, if that is not an issue to you, you can just ignore the warning.

**Reply by djnield** [reply]
Thanks! This is a me problem then.

Just on this one example I specified I can see the issue actually. Though there is over 7000 entries in the log to attempt fixing :(

I'll ask the seperatly the 2nd issue.

**Comment by SawkeeReemo** [comment]
Just gonna necro this old post for a sec: I'm getting these warnings now in 2024, but the episodes in question definitely are matched in Plex. Is there any other possible reason for this?

**Reply by TonioBDS** [reply]
Hello all, 
I'd like to share that I have a lot of similar issue with hundreds of warning in my PlexTraktSync log such as:
`WARNING  Blue Eye Samurai/s01e01/Episode 1 (2023): Skipping                     
         <PlexGuid:local://16826> because provider 'local' has no external Id`

In Plex, all my data seem to be fully well identified as the "correct media".
Does someone knows how the "external Id" is supposed to be identified by PlexTraktSync ?
Or anyone had and resolve the same issue?
Many thanks,

**Reply by glensc** [reply]
@SawkeeReemo: why do you say "definitely matched" and provide no proof?

run inspect on individual episodes and season and show. they may give mixed results, i.e, some episodes have matches while others have local.

ps: don't hijack discussions, create your own. you paste link to existing discussions if needed!

@TonioBDS if guid is "local", then plex has not made a match for it.

run `plextraktsync inspect 16826`, and it will show you guid is local

ps: don't hijack threads, create your own. you paste link to existing thread, if needed!

**Reply by SawkeeReemo** [reply]
> @SawkeeReemo: why do you say "definitely matched" and provide no proof?
> 
> run inspect on individual episodes and season and show. they may give mixed results, i.e, some episodes have matches while others have local.
>
I wasn't aware I needed to show proof for matched content as it's pretty obviously when something isn't matched in Plex. And I'd be happy to run `inspect` but I don't know what that is. Is that a function of PlexTraktSync that I'm not aware of? (about to jump back to the main page here to see if I can discern)
 
> ps: don't hijack discussions, create your own. you paste link to existing discussions if needed!

Sorry, I figured since it was the same issue, I'd keep from cluttering up the discussion topics. All good though, won't do this moving forward.

> 
> @TonioBDS if guid is "local", then plex has not made a match for it.
> 
> run `plextraktsync inspect 16826`, and it will show you guid is local
> 

Oh and now I see here that it is a function of PlexTraktSync. ~How exactly do you run that? I just use PTS as a docker container, so it runs, then exists.~ Never mind, figured it out.

**Reply by glensc** [reply]
inspect command is described in readme, if you need to improve it (documentation), feel free to send pull request. I'm really bad (and usually not interested) writing those docs. first lines of readme invite to contribute.

**Reply by SawkeeReemo** [reply]
Oh no worries at all. I figured it out. And even though it is properly listed in Plex, somehow one or two episodes of a season of this show are not officially “matched.” If I knew how to fix this with Plex, I’d go do that. I think the problems comes in when shows have an airing order and a DVD order… or when they weirdly have a season finale episode and for some weird reason it’s listed as a special. I don’t get it. ¯\_(ツ)_/¯ But everything with PlexTraktSync is working well! Just saved my ass from a server crash too! So thanks!

**Reply by glensc** [reply]
> it's pretty obviously when something isn't matched in Plex

here I strongly disagree. that plex shows metadata (texts) in web doesn't mean the match (still) exists. it could be copy of previous match, and somehow it got unmatched. and I have show that show page has guid (match), but seasons and episodes have guid local. the tmdb has those episodes data for a while already, but plex isn't able to pick it up somewhy.

**Reply by glensc** [reply]
The only thing you can do, is try to match show again (there's no match functionality for episodes). but be sure your files are named as trakt names, you may have to rename them. there's some section in readme about this. I can't comment that (much), because I didn't write that there.
